src/librustdoc/theme.rs

```diff
@@ -264,9 +264,11 @@ pub fn get_differences(against: &CssPath, other: &CssPath, v: &mut Vec<String>)
     }
 }
 
-pub fn test_theme_against<P: AsRef<Path>>(f: &P, against: &CssPath, diag: &Handler)
-    -> (bool, Vec<String>)
-{
+pub fn test_theme_against<P: AsRef<Path>>(
+    f: &P,
+    against: &CssPath,
+    diag: &Handler,
+) -> (bool, Vec<String>) {
```

Comment:
This reflowing seems to be spurious in terms of this patch.  Is it a result of `rustfmt` being run and if so, should it be here?  I don't know the policy on unrelated changes in this codebase.

Reply:
No, just me when reading code. A bad habit. In small PRs such as this one, we generally just don't look at it.

Reply:
Okay, I'll consider this resolved.